src/OpcApplicationConfiguration.cs

```diff
@@ -240,8 +242,7 @@ private static void ConfigureUserTokenPolicies(IApplicationConfigurationBuilderS
 
     private const int MAX_MESSAGE_QUEUE_SIZE = 200000;
     private const int MAX_NOTIFICATIONS_PER_PUBLISH = 200000;
-    private const int MAX_SUBSCRIPTION_COUNT = 200;
-    private const int MAX_PUBLISH_REQUEST_COUNT = MAX_SUBSCRIPTION_COUNT;
+    private const int MAX_PUBLISH_REQUEST_COUNT = 200;

```

Comment:
>200 [](http://example.com/codeflow?start=50&length=3)

The readme says 2000 and here says 200. Which is correct?

Reply:
The change and readme is about queued request count. That is different from publish request count.